communication/moderators.md

```diff
@@ -51,10 +55,13 @@ administration, but also moderation of issues, PRs, etc.
 
 Administrators seats: 4
 
-- Paris Pittman (@parispittman) - PT
-- Jorge Castro (@castrojo) - ET
-- Bob Killen (@mrbobbytables) - ET
-- Jeffrey Sica (@jeefy) - ET
+| Name                | Kubernetes Slack ID | Timezone                                                | UTC offset |
```

Comment:
It might be better to be their Discuss User ID -- the only one that would need to be changed is paris's to @parispittman